apps/lasp/src/lasp_core.erl

```diff
@@ -337,6 +337,26 @@ bind(Id, Value, Store) ->
 
 %% @doc Define a dataflow variable to be bound a value.
 -spec bind(id(), value(), function(), store()) -> {ok, var()}.
+bind(Id, {delta, Value}, MetadataFun, Store) ->
```

Comment:
bind() with {delta, Value} returns the delta state (Value) not the entire state (Merged) (see line 350). update() for the broadcast backend is using the state twice (one for the broadcast and the other for the return value). For the delta-mutation, the broadcast should use the delta state, but I am not sure about the return value of update(). Which state is the right one for the return value of update() with the delta-mutation (the delta state or the entire state)?